src/main/java/pageUrl/PageUrl.java

```diff
@@ -0,0 +1,5 @@
+package pageUrl;
+
+public class PageUrl {
+    public static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
```

Comment:
Following url is not used in:
Please specify here account and login page urls defined in LoginFormTest, LoginPageTest and CreateAccountTest and use them in mentioned test classes.